types/rebass/index.d.ts

```diff
@@ -66,7 +66,7 @@ export interface BoxProps
         WidthProps,
         FontSizeProps,
         RebassHTMLAttributes<HTMLDivElement> {}
-type BoxClass = StyledComponentClass<BoxProps, {}>
+type BoxClass = React.StatelessComponent<ThemedOuterStyledProps<BoxProps, {}>>
```

Comment:
Why the change from `StyledComponentClass`? 🤔

Reply:
Because a `StyledComponentClass` has a `withComponent` method and rebass components have not. 

If you take a look at `StyledComponentClass` you'll notice it extends `React.ComponentClass<ThemedOuterStyledProps<O, T>>` which, when instantiated, returns a `React.Component<ThemedOuterStyledProps<O, T>>`. Moreover, I don't expect people to use any of the `React.ComponentClass` attributes, if I'm not mistaken, those would be for setting propTypes, defaultProps, etc. on rebass components which would affect all instances. I am not seeing the use case for doing that.

I also favored using `React.StatelessComponent` instead of `React.Component` because I don't think we need the `setState`, `forceUpdate`, etc. methods on rebass components.